examples/camera.py

```diff
@@ -91,7 +91,9 @@ def main(self):
 
             self.get_and_flip()
             self.clock.tick()
-            pygame.display.set_caption("CAMERA! (" + str(round(self.clock.get_fps())) + " FPS)")
+            pygame.display.set_caption(
+                "CAMERA! (" + str(round(self.clock.get_fps())) + " FPS)"
```

Comment:
This could become an f-string for better readability, something like `f"CAMERA! ({self.clock.get_fps():.2f} FPS)"` where `.2f` rounds the float to 2 decimal places